src/diffusers/pipelines/controlnet/pipeline_controlnet_inpaint.py

```diff
@@ -1264,6 +1298,17 @@ def __call__(
         else:
             batch_size = prompt_embeds.shape[0]
 
+        if padding_mask_crop is not None:
+            if width is None or height is None:
+                default_height, default_width = self.image_processor.get_default_height_width(image)
+                width = width or default_width
+                height = height or default_height
```

Comment:
```suggestion
            height, width = self.image_processor.get_default_height_width(image, height, width)
```

if you look at the code of `get_default_height_width()` - it automatically handles the situation where `height` or `width` is `None